modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/util/CustomDateDeserializer.java

```diff
@@ -0,0 +1,22 @@
+package org.motechproject.scheduler.util;
+
+import org.codehaus.jackson.JsonParser;
+import org.codehaus.jackson.JsonProcessingException;
+import org.codehaus.jackson.map.DeserializationContext;
+import org.codehaus.jackson.map.JsonDeserializer;
+import org.codehaus.jackson.map.JsonMappingException;
+import org.joda.time.DateTime;
+
+import java.io.IOException;
+
+public class CustomDateDeserializer extends JsonDeserializer<DateTime> {
```

Comment:
Don't see a point in custom serializing/deserializing, but javadocs are missing in all these classes